client/src/util/ViewCache.res

```diff
@@ -1,5 +1,6 @@
 open Tc
 
+// WHAT do we change this often? ever? maybe this should be a dev setting?
```

Comment:
We change this very rarely, only to debug view caching errors. No point exposing it imo.